pkg/reporter/summary.go

```diff
@@ -262,6 +266,12 @@ func (r *summaryReporter) processForMalware(pkg *models.Package) {
 	}
 }
 
+func (r *summaryReporter) processForDepsUsageEvidence(pkg *models.Package) {
+	if pkg.CodeAnalysis == nil || len(pkg.CodeAnalysis.UsageEvidences) == 0 {
```

Comment:
This is probably misleading. Consider the case where `--code` is not enabled during scan. Also there are cases of transitive dependencies. We are only looking at App (1st party) code so we cannot be certain that a transitive dependency is used or not.

I think we should introduce:

- `used`
- `unknown`

Track `used` when we have usage evidence. Else unknown.

Reply:
yep, makes sense